docusaurus/docs/reactnative/basics/troubleshooting.mdx

```diff
@@ -209,6 +209,16 @@ android {
 }
 ```
 
+## External Link not opening
+
+### Android
+
+Opening an external link doesn't work without permissions on `targetSdkVersion` >= 30. So, the following permission must be included in the `AndroidManifest.xml` file.
+
+```xml
+<uses-permission android:name="android.permission.QUERY_ALL_PACKAGES" />
+```
+
```

Comment:
```suggestion
### Android

For applications targeting Android 11 (API Level 30, `targetSdkVersion=30`), Android has introduced changes to package visibility. If a link clicked in your application tries to navigate the end user to an application that yours can't see as a result of this will therefore lead to nothing happening.

To get around this, the following permission change can be included in your `AndroidManifest.xml`:

```xml
<uses-permission android:name="android.permission.QUERY_ALL_PACKAGES" />
```
:::caution
**Note:** Make sure you are aware of [upcoming policies](https://support.google.com/googleplay/android-developer/answer/10158779) that this change will be subject to.
:::
```

I got curious about this, and read a bit of the docs for it, and I think it's worth adding a bit more information about why this is needed, especially since it will affect whether their app can be published on Google Play or not in the future.

My wording here is just a suggestion though, feel free to rewrite entirely. But I think we should have some more information about it if we include it in here.

Reply:
Agh, okay :)

Reply:
I followed the pattern the others were following, didn't think much about it, and I don't think if we mention `targetSdkVersion` we need to care much about the details, but anyways thanks for your suggestions. :)

Reply:
```
:::caution
**Note:** Make sure you are aware of [upcoming policies](https://support.google.com/googleplay/android-developer/answer/10158779) that this change will be subject to.
:::
```

This should be a good enough addition I think.